### PR TITLE
[ADD] json Scenario route resolve #3, #4, #2

### LINKED ISSUE #3: Create arango db pool
We use arangodb to save scenarrous, we need a pool of connection to use this in our dao.

### PR DESCRIPTION
**Pull request by kostya05983**

### CODE REVIEW

File: api/src/main/kotlin/bomber/arango/ArangoDao/JsonScenarioDao.kt [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+import com.arangodb.model.AqlQueryOptions
+import com.arangodb.util.MapBuilder
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.module.kotlin.readValue
+
+/**
+ * This dao is responsible for crud operation scenarious
+ * @author kostya05983
+ */
+class JsonScenarioDao {
```
Comment (kubitre) [inline api/src/main/kotlin/bomber/arango/ArangoDao/JsonScenarioDao.kt:16]: maybe we will call json in uppercase as JSON. Identical similar objects: IP, DAO, ID,

File: api/src/main/kotlin/bomber/arango/ArangoDao/JsonScenarioDao.kt [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    private val objectMapper = ObjectMapper()
+
+    init {
+        arangoDb.createDatabase(DB_NAME).get()
+        db = arangoDb.db(DB_NAME)
+        db.createCollection(COLLECTION_NAME).get()
+        collection = db.collection(COLLECTION_NAME)
+    }
+
+    private enum class VarFields(val text: String) {
```
Comment (kubitre) [inline api/src/main/kotlin/bomber/arango/ArangoDao/JsonScenarioDao.kt:36]: how it works?

Comment (kubitre) [reply api/src/main/kotlin/bomber/arango/ArangoDao/JsonScenarioDao.kt:36]: aa, i'm understand, it is simple way for constant declaring=) Am i right?

File: api/src/main/kotlin/bomber/arango/ArangoDao/JsonScenarioDao.kt [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+    /**
+     * Down our dao for scenarious interrupt connections and so on
+     */
+    fun downDao() {
+        db.drop().get()
+        arangoDb.shutdown()
+    }
+
+    /**
+     * Get scenario by id
```
Comment (kubitre) [inline api/src/main/kotlin/bomber/arango/ArangoDao/JsonScenarioDao.kt:50]: you should finish this comment to full

**Review by kubitre — APPROVED** [review]

File: api/src/main/kotlin/bomber/arango/ArangoDao/JsonScenarioDao.kt (hunk 2 above)
Comment (kostya05983) [reply api/src/main/kotlin/bomber/arango/ArangoDao/JsonScenarioDao.kt:36]: Yes, you are right, It is used for constant.

File: api/src/main/kotlin/bomber/arango/ArangoDao/JsonScenarioDao.kt (hunk 3 above)
Comment (kostya05983) [reply api/src/main/kotlin/bomber/arango/ArangoDao/JsonScenarioDao.kt:50]: I will finish it.

File: api/src/main/kotlin/bomber/arango/ArangoDao/JsonScenarioDao.kt (hunk 1 above)
Comment (kostya05983) [reply api/src/main/kotlin/bomber/arango/ArangoDao/JsonScenarioDao.kt:16]: I can this rename,and use it as the name of class, but in java we rarely call json as JSON , but  I rename Dao in DAO

**Pull request merged**